### ISSUE TITLE
Evolution Strategies as a Scalable Alternative to Reinforcement Learning

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/nn-algorithm#70]
# Evolution Strategies as a Scalable Alternative to Reinforcement Learning #

- Author: OpenAI, Tim Salimans, Jonathan Ho, Xi Chen, Ilya Sutskever
- Origin: https://blog.openai.com/evolution-strategies/
- Related:
  - [arxiv](https://arxiv.org/abs/1703.03864)
  - https://github.com/openai/evolution-strategies-starter
  - [Evolution Strategies: Almost Embarrassingly Parallel Optimization](http://www.inference.vc/evolutionary-strategies-embarrassingly-parallelizable-optimization/)
  - [木刻思: Reinforcement Learning without Backprop:Open AI Evolution Strategies](http://data-sci.info/2017/04/01/reinforcement-learning-without-backpropopen-ai-evolution-strateies/)
  - [Soumith Chintala's twitter](https://twitter.com/soumithchintala/status/845349987957510145)
  - [hardmaru's twitter](https://twitter.com/hardmaru/status/846122381319925760)

> ”Previously, very good things have come out of researchers questioning fundamental assumptions about our methods: If you think about it, replacing smooth sigmoid activations with non-differentiable rectified linear units sounds like a pretty bad idea – until you actually realise they work. Dropout may sound like something to be avoided, until you realise why it works….So ES may well be one of these things. Salimans et al questioned a fundamental thing that everybody takes for granted: that neural networks are optimised via SGD using backpropagation. Well, maybe not always.”
